ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Overview/CVEsTable.tsx

```diff
@@ -195,6 +195,7 @@ function CVEsTable({
                                             moderateCount={moderate.total}
                                             lowCount={low.total}
                                             filteredSeverities={filteredSeverities}
+                                            entity={'node'}
```

Comment:
A missed attribute during development - the tooltip for these by default uses "image" as the entity type.